fix from upstream
http://bugs.gentoo.org/300909

r26689 | jmarshallnz | 2010-01-11 14:30:08 -0500 (Mon, 11 Jan 2010) | 2 lines                                                                                               
fixed: Ticket #7810 - high cpu load during loading of images with libjpeg7, thanks to akawaka.

Index: xbmc/lib/cximage-6.0/CxImage/ximajpg.cpp
===================================================================
--- xbmc/lib/cximage-6.0/CxImage/ximajpg.cpp	(revision 26688)
+++ xbmc/lib/cximage-6.0/CxImage/ximajpg.cpp	(revision 26689)
@@ -220,6 +220,7 @@ bool CxImageJPG::Decode(CxFile * hFile)
 
 	// Set the scale <ignacio>
 	cinfo.scale_denom = GetJpegScale();
+	cinfo.scale_num = 1;
 
 	// Borrowed the idea from GIF implementation <ignacio>
 	if (info.nEscape == -1) {
